src/Sylius/Bundle/CoreBundle/Resources/config/sylius.yml

```diff
@@ -138,101 +39,122 @@ sylius_archetype:
             attribute: Sylius\Component\Product\Model\Attribute
             option: Sylius\Component\Product\Model\Option
 
-sylius_attribute: ~
-
-sylius_variation:
+sylius_association:
     resources:
         product:
-            variant:
+            subject: "%sylius.model.product.class%"
+            association:
                 classes:
-                    model: Sylius\Component\Core\Model\ProductVariant
-                    repository: Sylius\Bundle\CoreBundle\Doctrine\ORM\ProductVariantRepository
-                    form:
-                        default: Sylius\Bundle\CoreBundle\Form\Type\ProductVariantType
+                    model: Sylius\Component\Product\Model\ProductAssociation
+                    interface: Sylius\Component\Product\Model\ProductAssociationInterface
```

Comment:
And probably with the bundle ([composer.json](https://github.com/michalmarcinkowski/Sylius/blob/f2ef70f73186b6ca73d658abde1d5310c8fcf270/src/Sylius/Bundle/ProductBundle/composer.json#L29-L29)).